static/src/javascripts/projects/commercial/modules/third-party-tags/facebook-pixel.js

```diff
@@ -1,19 +1,10 @@
 // @flow
 import config from 'lib/config';
-import {
-    getAdConsentState,
-    thirdPartyTrackingAdConsent,
-} from 'common/modules/commercial/ad-prefs.lib';
 
-export const fbPixel: () => ThirdPartyTag = () => {
-    const consent = getAdConsentState(thirdPartyTrackingAdConsent);
-    return {
-        shouldRun:
-            config.get('switches.facebookTrackingPixel') &&
-            (consent == null || consent),
-        url: `https://www.facebook.com/tr?id=${config.get(
-            'libs.facebookAccountId'
-        )}&ev=PageView&noscript=1`,
-        useImage: true,
-    };
-};
+export const fbPixel: () => ThirdPartyTag = () => ({
```

Comment:
Does this remove the existing checks to prevent this tag from showing up based on consent signals from the current CMP?

Reply:
It removes it here because we introduced a check at the third-party-tags,js level (which is what calls facebook-pixel.js)